### DISCUSSION TITLE
Artisan 3.0 button labels \v \f \t

### DISCUSSION THREAD

**Discussion by hankua** [artisan-roaster-scope/artisan#1646] [Q&A]
How are the updated button labels \v \f \t used?
“ adds additional button label substitutions, \V, \F and \T reporting the event value, the event value interpreted as temperature in Fahrenheit, and the event value interpreted as temperature in Celsius. The last two are automatically converted to the currently selected temperature unit.”

**Comment by roasterdave** [comment]
Here is an example shown in both Fahrenheit and Celsius modes.

Fahrenheit mode
![buttons f 002](https://github.com/user-attachments/assets/f6f4bb1c-0477-4c0a-b428-bf02802f53bd)

Celsius mode
![buttons c](https://github.com/user-attachments/assets/a3ba913a-0c57-40e2-8f12-81e3a831e05a)

**Reply by hankua** [reply]
Thanks!  Not sure how this is used in practice; charge, TP, EOD, and FC already mark. If EOD is 160c, then special button for 170c and 180c?

**Reply by MAKOMO** [reply]
There are machine setups which send a certain set value (SV) temperature to the connected device. Those setups can now be organized to work in C and F always displaying the correct label on the buttons according to the selected temperature mode.
